src/Microsoft.AspNetCore.Razor.Language/Intermediate/TagHelperIntermediateNode.cs

```diff
@@ -49,6 +50,30 @@ public override RazorDiagnosticCollection Diagnostics
 
         public ICollection<TagHelperDescriptor> TagHelpers { get; } = new List<TagHelperDescriptor>();
 
+        public TagHelperBodyIntermediateNode Body
+        {
+            get
+            {
+                return Children.OfType<TagHelperBodyIntermediateNode>().SingleOrDefault();
+            }
+        }
+
+        public IReadOnlyCollection<SetTagHelperPropertyIntermediateNode> SetTagHelperProperties
```

Comment:
Since we decided to rename these nodes, I did not put a lot of thought in to the property name.